libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -41,7 +41,7 @@ Implemented Papers
 - P2637R3 - Member ``visit``
 - P2652R2 - Disallow User Specialization of ``allocator_traits``
 - P2819R2 - Add ``tuple`` protocol to ``complex``
-- P2495R3 - Interfacing ``stringstream``s with ``string_view``
+- P2495R3 - Interfacing ``stringstreams`` with ``string_view``
```

Comment:
```suggestion
- P2495R3 - Interfacing ``stringstream``\ s with ``string_view``
```

Reply:
Thank you a lot! I wonder why there isn't an issue here:
https://github.com/llvm/llvm-project/blob/762f762504967efbe159db5c737154b989afc9bb/libcxx/docs/Status/SpaceshipPapers.csv?plain=1#L3

Reply:
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#character-level-inline-markup-1
The white space shouldn't be necessary. This should be enough:
> \``stringstream\``\s